app/partials/states/account_settings.html

```diff
@@ -295,16 +295,19 @@ <h2 class='subheader'>Editing another user's Isaac account.</h2>
                                     <strong ng-if="!user.hasSegueAccount">Set password</strong>
                                 </label>
                                 <!-- Can't set new password if not confirmed old one, or if don't have account with a password. Unless admin. -->
-                                <input id="account-password" ng-disabled="(!passwordChangeState.passwordCurrent || !user.hasSegueAccount)  && editingSelf" type="password" ng-model="password1" name="password" placeholder="Enter password">        
+                                <input id="account-password" ng-disabled="(!passwordChangeState.passwordCurrent || !user.hasSegueAccount)  && editingSelf" type="password" ng-model="password1" name="password" placeholder="Enter password" ng-class="{'ng-invalid': password1.length < 6}">
                             </div>
                         </div>
                         <div id="password2-wrap" class="row">
                             <div  class="large-6 columns">
-                                <label for="account-password2" ng-class="{'disabled' : !password1}"><strong>Confirm password</strong></label>
-                                <input id="account-password2" type="password" placeholder="Re-enter password" ng-disabled="!password1" match="password1" ng-model="password2" name="password2">
+                                <label for="account-password2" ng-class="{'disabled' : (!password1 || password1.length < 6)}"><strong>Confirm password</strong></label>
+                                <input id="account-password2" type="password" placeholder="Re-enter password" ng-disabled="(!password1 || password1.length < 6)" match="password1" ng-model="password2" name="password2">
                                 <span class="ru-error-message" ng-if="(password1 || password2) && !passwordChangeState.passwordCurrent && editingSelf">
                                     Current password not provided
                                 </span>
+                                <span class="ru-error-message" ng-if="password1.length < 6">
+                                    Password must be at least 6 characters long
+                                </span>
```

Comment:
Small CSS suggestion - 
Might be worth having a quick play to see if there is a nicer column width for the section/error messages. The minimum length error message, on my screen anyway, wraps around where it probably doesn't need to.  
If it looks odd with "large-8 columns" rather than large-6 like we have on other tabs then ignore this suggestion.